### DISCUSSION TITLE
child window&owner window

### DISCUSSION THREAD

**Discussion by ggslayer** [tauri-apps/tauri#8574] [Q&A] [answered]
Hi, I want to know the different between child window and owned window. 
In my requirements:I want the child window have importent features:
1. the "child window"'s position is fixed in it's parent, can't move everywhere, unless I reset the position.
2. child window not show in system's taskbar

I don't know which should I select.

and then.. I see the macos only support "chlid", the linux support none of them?   will they be supported in future?

**Comment by FabianLars** [comment, marked as answer]
1. if i recall correctly you want the child/parent window relation ship. the child will be confined to the parent window (if it's bigger then it will be clipped like `overflow: hidden`
2. I'm not exactly sure how a child window is handled, but you could try the skipTaskbar config if it shows up

> I see the macos only support "chlid"

Yep, it doesn't have the 2 concepts Windows (OS) has.

> the linux support none of them?

I think on Linux it also has the child behavior but no owner api. It's implemented in tao (tauri's windowing library) but was never exposed in tauri for some reason. Should be an easy change though.

**Reply by ggslayer** [reply]
Ok, Thank you, Is there any plan to open child window feature on linux near future?     My app need it

**Reply by FabianLars** [reply]
Could you open a feature request (a normal github issue) so we don't forget about it please?

**Reply by ggslayer** [reply]
ok  thanks
